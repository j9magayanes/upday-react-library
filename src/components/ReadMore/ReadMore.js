import React from 'react';
import './readmore.css';


export const ReadMore= ({args}) => {
        return (
            <div className="page">
            <article>
            <div className="padded-lr legal">
                <h1 id="imprint">{args.detail}</h1>
            </div>
            </article>
            </div>)}