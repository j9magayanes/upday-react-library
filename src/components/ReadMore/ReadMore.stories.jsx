import React from 'react';


import { ReadMore } from './ReadMore';

export default {
  title: 'Form/ReadMore',
  component: ReadMore,
};

const Template = (args) => <ReadMore {...args} />;

export const Web = Template.bind({});
Web.args = {
  label: 'Web',
  args:{  text: 'Read More', id: 'web'}
};

export const Meinestadt = Template.bind({});
Meinestadt.args = {
  label: 'Default',
  args:{  text: 'Read More', id: 'meinestadt'}
};