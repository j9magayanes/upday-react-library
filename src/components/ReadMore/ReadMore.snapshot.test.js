// ReadMore.test.js

import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import { ReadMore } from './ReadMore';

test('renders ReadMore component correctly', () => {
  const args = {
    id: '123',
    text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
  };

  const { container } = render(<ReadMore args={args} />);
  expect(container).toMatchSnapshot();
});
