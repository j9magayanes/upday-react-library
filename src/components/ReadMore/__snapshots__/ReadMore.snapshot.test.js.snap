// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ReadMore component correctly 1`] = `
<div>
  <span
    id="storybook-read-more-123"
  >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
  </span>
</div>
`;
